WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuFragment.kt

```diff
@@ -194,13 +180,32 @@ class MoreMenuFragment : TopLevelFragment() {
         )
     }
 
+    private fun openGoogleForWooWebview(url: String, canAutoLogin: Boolean) {
+        if (canAutoLogin) {
+            openInAuthBrowser(url)
+        } else {
+            openInExitAwareWebview(url)
+        }
```

Comment:
Np, and probably personal preference but it usually looks nicer and less lines of code to use when style. No need to change anything, just leaving the suggestion.
```suggestion
when {
    canAutoLogin -> openInAuthBrowser(url)
    else -> openInExitAwareWebview(url)
}
```

Reply:
Yup, looks better.